aws/resource_aws_glue_crawler.go

```diff
@@ -455,29 +458,49 @@ func resourceAwsGlueCrawlerUpdate(d *schema.ResourceData, meta interface{}) erro
 	glueConn := meta.(*AWSClient).glueconn
 	name := d.Get("name").(string)
 
-	updateCrawlerInput, err := updateCrawlerInput(name, d)
```

Comment:
The original update logic was only indented with the addition of the `if` conditional (not sure why the Git diff looks so weird).